SensusService/Probes/User/Scripts/ScriptRunner.cs

```diff
@@ -415,52 +446,37 @@ private void StartRandomTriggerCallbacks()
 #error "Unrecognized platform."
 #endif
 
-                // find next future trigger window, ignoring month, day, and year of windows. the windows are already sorted.
+                // find number of days left in participation horizon
```

Comment:
You're right, I misunderstood. We discussed immediately scheduling all scripts for the duration of a study in our meeting last week and I thought `ParticipationHorizon` established an upper bound on study duration. Should we schedule scripts on a day by day basis instead?